src/ExpressionEvaluator/Core/Source/ExpressionCompiler/MetadataUtilities.cs

```diff
@@ -105,42 +114,98 @@ internal static partial class MetadataUtilities
                 modulesByName[name] = modulesByName.ContainsKey(name) ? null : metadata;
             }
 
+            // We don't walk the references of winmd assemblies currently. (That would require
+            // walking references from the current module and also the winmd assemblies.)
+            // So if there are any winmd assemblies, we'll use all assemblies.
```

Comment:
>So if there are any winmd assemblies, we'll use all assemblies. [](start = 15, length = 63)

Should we have an issue for this? We might want to address this in 15.8

Reply:
Logged #26157.

---
In reply to: [181510450](https://github.com/dotnet/roslyn/pull/25955#discussion_r181510450) [](ancestors = 181510450)